### DISCUSSION TITLE
Clue #2

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#236] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
![Screenshot 2022-11-14 8 36 54 AM](https://user-images.githubusercontent.com/77510164/201674215-bd5b4f71-516f-44f8-86a3-7bf5f83831f6.png)
As pictured above...
A naïve google accidentally shows us a vulnerability- it's cache. It also shows how policies aren't constantly checked.

**Comment by ILikeBypassing** [comment]
so how long are these intervals exactly

**Reply by ghost** [reply]
Nobody knows or cares tbh

**Comment by Aka-but-nice** [comment]
Yall can find XSS vulnerabilities in, almost any extension then use them to inject HTML basically swamp but for any extension
